telegram/ext/filters.py

```diff
@@ -324,6 +324,65 @@ def filter(self, message):
 
     group = _Group()
 
+    class user(BaseFilter):
```

Comment:
I know about it, but in the same file we have classes, which names are not CamelCase (for example, `language` class below)